backend/codes/models.py

```diff
@@ -29,10 +29,16 @@ def __str__(self):
         return f'Code {self.id} {self.name}'
 
 
-class PriorityQueue(models.Model):
+class AnimationPriorityQueueElement(models.Model):
     code = models.ForeignKey(Code, on_delete=models.CASCADE)
     priority = models.PositiveIntegerField(default=0, validators=[MinValueValidator(0), MaxValueValidator(10)])
 
+    def pop_first():
+        highest_priority_element = AnimationPriorityQueueElement.objects.order_by('-priority').first()
+        AnimationPriorityQueueElement.objects.order_by('-priority').first().delete() 
```

Comment:
Nie można po prostu
```
highest_priority_element.delete()
```
? w linii 38

Reply:
tylko że highest_priority_element jest zwracany w tej funkcji, jak nie trzeba zwracać to może tak być jak napisałeś